src/routes/upcoming/+page.svelte

```diff
@@ -86,7 +88,7 @@
 		</article>
 	{/each}
 
-	{#each data.upcomingEvents as { speakers, title, date, eventUrl }}
+	<!-- {#each data.upcomingEvents as { speakers, title, date, eventUrl }}
```

Comment:
Should we just delete the contents of this comment?

Reply:
Yeah, must have missed it, I deleted a ton.